packages/components/src/modifiers/hds-code-editor.ts

```diff
@@ -239,15 +256,47 @@ export default class HdsCodeEditorModifier extends Modifier<HdsCodeEditorSignatu
 
   private _setupEditorAriaDescribedBy(
     editor: EditorViewType,
-    ariaDescribedBy?: string
+    {
+      ariaDescribedBy,
+      lintingDescriptionElement,
+    }: {
+      ariaDescribedBy?: string;
+      lintingDescriptionElement?: HTMLParagraphElement;
+    }
   ) {
-    if (ariaDescribedBy === undefined) {
+    if (
+      ariaDescribedBy === undefined &&
+      lintingDescriptionElement === undefined
+    ) {
       return;
     }
 
+    const ariaDescribedByArray = [];
+
+    if (ariaDescribedBy !== undefined) {
+      ariaDescribedByArray.push(ariaDescribedBy);
+    }
+
+    if (lintingDescriptionElement !== undefined) {
+      ariaDescribedByArray.push(lintingDescriptionElement.id);
+    }
+
     editor.dom
       .querySelector('[role="textbox"]')
-      ?.setAttribute('aria-describedby', ariaDescribedBy);
+      ?.setAttribute('aria-describedby', ariaDescribedByArray.join(' '));
+  }
+
+  private _createLintingDescriptionElement(): HTMLParagraphElement {
+    const element = document.createElement('p');
+
+    element.id = `lint-panel-instructions-${this.element.id}`;
+    element.classList.add('sr-only');
+    element.textContent =
+      'Press `Ctrl-Shift-m` (`Cmd-Shift-m` on macOS) to open the linting panel';
```

Comment:
```suggestion
      'Press `Ctrl-Shift-m` (`Cmd-Shift-m` on macOS) while focus is on the textbox to open the linting panel';
```

Suggestion: Just to be slightly more specific.